examples/simple/app.js

```diff
@@ -1,9 +1,94 @@
-/*global DataSource, FastList*/
+/*global FastList*/
+'use strict';
+
+var debug = 0 ? (...args) => console.log('[DataSource]', ...args) : ()=>{};
 
 var content = document.getElementById('content');
 var data = createDummyData(1000);
-var dataSource = new DataSource(data);
-var list = new FastList(content, dataSource);
+var itemHeight = 88;
+
+var list = new FastList({
+  container: content,
+  itemTemplate: '<li><h3> </h3><p> </p></li>',
+  sectionTemplate: '<section><h2> </h2><div class="background"></div></section>',
+
+  populateItem: function(el, i) {
+    debug('populate item', el);
+    var title = el.firstChild;
+    var body = title.nextSibling;
+    var record = this.getRecordAt(i);
+
+    title.firstChild.data = record.name;
+    body.firstChild.data = record.metadata.artist;
+  },
+
+  populateSection: function(el, section, i) {
+    var title = el.firstChild;
+    var height = this.fullSectionHeight(section);
+    var background = title.nextSibling;
+
+    background.style.height = height + 'px';
+    title.firstChild.data = section;
+  },
+
+  getSections() {
+    debug('get sections');
+    return [];
```

Comment:
I think that is also a bug as we end up with `<h2>` being rendered when we don't need it. Could we instead apply the background to the `<ul>` and not render any unwanted sections?